crates/extensions/tedge_mqtt_ext/src/lib.rs

```diff
@@ -139,6 +147,7 @@ impl ToPeers {
         incoming_mqtt: &mut mpsc::UnboundedReceiver<MqttMessage>,
     ) -> Result<(), RuntimeError> {
         while let Some(message) = incoming_mqtt.next().await {
+            tracing::info!(target: "MQTT recv", "[{}] {}", &message.topic.name, message.payload_str().unwrap_or_default());
```

Comment:
This gives :

```
2024-06-26T15:35:33.230788346Z  INFO MQTT recv: [te/device/main/service/tedge-mapper-bridge-c8y/status/health] {"status":"up"}
```

to be compared with the current log entries:

```
2024-06-20T17:50:28.806953202Z DEBUG MQTT: send MqttMessage { topic: Topic { name: "te/device/main/service/tedge-mapper-bridge-c8y/status/health" }, payload: "{"status":"up"}", qos: AtLeastOnce, retain: true } 
```

Where we have to understand that the MQTT actor is sending to the other actors a message that has been just received from its connection to the broker.

Same questions as for the published messages: what is the appropriate log level?

Reply:
I think the log level depends on how often are these messages being sent around...for periodic data like the measurements/events/alarms, `DEBUG` is more appropriate, but maybe for services having the status updates shown on `INFO` level would be more hopeful to understanding the interactions between the services in the logs, though since we don't really want to go down the conditional log level (based on topics), is there a "health"status actor that can write the `INFO` log messages (rather than the mqtt actor)?